### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by btecu**
- Lint
- Remove some unused packages
- Remove archived package (`multiline`)
- Update packages

### CODE REVIEW

File: blueprints/addon-import/index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ module.exports = {
     let blueprintName = options.originBlueprintName;
     let modulePathSegments = [addonName, inflector.pluralize(options.originBlueprintName), fileName];
 
-    if (/-addon/.test(blueprintName)) {
+    if ((/-addon/).test(blueprintName)) {
```
Comment (kellyselden) [inline blueprints/addon-import/index.js]: We have a lint rule for this?

Comment (btecu) [reply blueprints/addon-import/index.js]: It might be one of the default rules https://eslint.org/docs/rules/wrap-regex.

Comment (kellyselden) [reply blueprints/addon-import/index.js]: I'm confused. If it's not already failing on master, then it must not be in our ruleset.

**Comment by Turbo87** [comment]
@btecu it looks like there is a merge conflict 😢

**Comment by btecu** [comment]
@Turbo87 Rebased.

**Comment by Turbo87** [comment]
thanks! :)

**Pull request merged**